src/components/Home/LogosSlider/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import cn from 'classnames'
+
+import * as styles from './styles.module.css'
+import CompanyLogos from './CompanyLogos'
+
+const CompanySlider = () => {
+  return (
+    <div className="mt-10">
+      <div className="mb-5">
+        <h2 className={cn('text-2xl font-medium text-center', styles.title)}>
+          Empowering from startups to Fortune 500 Companies: Our Open Source
+          Tools at Work
```

Comment:
Suggest this:
> Empowering thousands of users and customers from startups to fortune 500 companies

notes:
- avoiding open source mention - some use FOSS, some paid, and future rebranding might make this more misleading, so keeping it general
- sentence case (not title case)
- don't think we need ":" here (also avoided ",") - will look cleaner imo

Reply:
@omesser 
>avoiding open source mention - some use FOSS, some paid, and future rebranding might make this more misleading, so keeping it general

Just for context: We have on purpose made the dvc.org website about open source (see hero). 
All of these logos will not be able to be used for Studio. They can only be used for DVC website (or eventual page in rebranding) except for the ones that came from the Iterative site. So the open-source part is not an issue here.

That being said, I like your edit.  Because open-source is mentioned in the hero, this simplifies the idea into one thought: EMPOWER, and lets people reflect on logos. I like it.

Reply:
> We have on purpose made the dvc.org website about open source (see hero).

Yep, that makes total sense @jendefig  - makes sense IMO to refer here to both users and customers, in the page-context of open source like you mentioned!